cvat-ui/src/actions/useragreements-actions.ts

```diff
@@ -30,8 +31,13 @@ export const getUserAgreementsAsync = (): ThunkAction => async (dispatch): Promi
     dispatch(userAgreementsActions.getUserAgreements());
 
     try {
-        const userAgreements = await core.server.userAgreements();
-        dispatch(userAgreementsActions.getUserAgreementsSuccess(userAgreements));
+        const isExists = await isReachable(`${core.config.backendAPI}/user-agreements`, 'OPTIONS');
```

Comment:
I don't think that we should access an URL outside of cvat-core

Reply:
the logic moved to cvat-core